### ISSUE TITLE
[admin] Show readonly id of organization

### ISSUE THREAD

**Issue by R9295** [openwisp/openwisp-radius#23]
**Reserved for GCI**

In order to authenticate with the API, you need to know the organization's id and it's token. The token is displayed, but the org id however, is not. Create a **read-only** field  in ```OrganizationAdmin``` that displays the org's id.

**Comment by nemesisdesign** [comment]
Something similar is implemented in other modules of openwisp:

- look for `id_hex` here: https://github.com/openwisp/django-netjsonconfig/blob/master/django_netjsonconfig/base/admin.py#L263-L280
- this javascript enhances the UX: https://github.com/openwisp/django-netjsonconfig/blob/master/django_netjsonconfig/static/django-netjsonconfig/js/uuid.js, needs to be loaded this way: https://github.com/openwisp/django-netjsonconfig/blob/master/django_netjsonconfig/base/admin.py#L44

**Comment by nemesisdesign** [comment]
@R9295 do we have this added into the GCI system as a task yet?

**Comment by R9295** [comment]
@nemesisdesign, just added it

**Comment by OltarzewskiK** [comment]
`OrganizationAdmin` is feature of `openwisp-users`, So can I add the read-only in `openwisp-users`?

**Comment by OltarzewskiK** [comment]
ok, I was thinking about it, but better make sure and do not make mistakes

**Comment by R9295** [comment]
Yes, see how the attributes are modified here: https://github.com/openwisp/openwisp-radius/blob/master/openwisp_radius/admin.py#L150.

**Comment by OltarzewskiK** [comment]
@R9295 Unfortunately your idea doesn't work, because `OrganizationAdmin` doesn't have `fields` feature

**Comment by R9295** [comment]
@OltarzewskiK Yes it does. Have you tried ```OrganizationAdmin.readonly_fields += ('id', )```?

**Comment by OltarzewskiK** [comment]
@R9295 :man_facepalming: it works, I was trying to add `OrganizationAdmin.fields += ('id',)`
Actually you solved my task for me

**Comment by R9295** [comment]
There is more to do! I think the ID should be visible on top, so figure out how to insert it at the beginning of the tuple.

**Comment by OltarzewskiK** [comment]
Sure!

**Comment by OltarzewskiK** [comment]
@R9295 `id` is only element in `OrganizationAdmin.readonly_fields` tuple, so `'id'` is in the beginning.

**Comment by R9295** [comment]
@nemesisdesign what do you suggest?

**Comment by nemesisdesign** [comment]
Hey guys, I think it may be better to do this change directly in `openwisp-users`. It would be easier and it's acceptable because I'm sure other modules will need a similar thing in the future, so let's make it there directly.

@R9295 @atb00ker do you see any problem with that?

PS: we may have a chance to try the new github feature to move an issue to a different repo :blush:

**Comment by OltarzewskiK** [comment]
@nemesisdesign I have only added this line: `readonly_fields = ['id']` coverage is the same. Is it enough?
